pkgs/development/tools/haskell/vaultenv/default.nix

```diff
@@ -96,8 +98,4 @@ mkDerivation rec {
   description = "Runs processes with secrets from HashiCorp Vault";
   license = lib.licenses.bsd3;
   maintainers = with lib.maintainers; [ lnl7 manveru ];
```

Comment:
Could you retarget this PR to the `haskell-updates` branch?  We generally ask all Haskell-related PRs to be targeted to that branch.

Also, if you use `vaultenv`, maybe you'd consider adding yourself as a maintainer here?

Reply:
> Could you retarget this PR to the `haskell-updates` branch? We generally ask all Haskell-related PRs to be targeted to that branch.

Done.

> Also, if you use `vaultenv`, maybe you'd consider adding yourself as a maintainer here?

No, sorry, I don't have the time to be an official maintainer.